### PR TITLE
Issue 28

### PR DESCRIPTION
**Pull request by marouaneamz**
piercus #28

### CODE REVIEW

**Comment by piercus** [comment]
@marouaneamz Travis fails, can you check why ?

**Comment by marouaneamz** [comment]
@piercus  travis failed unexpectedly without any error message and I find that the repository url in package.json should end with git, which it does, otherwise you need to delete the cache key from your .travis. yml.

**Comment by piercus** [comment]
@marouaneamz 

see in https://app.travis-ci.com/github/alezanai/object-tracking-measure/builds/253099084

```
  tests/issue-28.js:15:8
220  ✖  15:8  result is assigned a value but never used.  no-unused-vars
```

There is a linting issue.

You can test your linting locally before pushing by running `npx xo` 
Xo is able also to fix most of errors by doing `npx xo --fix`

**Comment by marouaneamz** [comment]
@piercus 
I solved the problem

File: tests/issue-28.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const otm = require('..');
+const test = require('ava');
+const groundTruths = [
+
+	[0, 1, 2, 3, 4] // X, y, w, h
+
```
Comment (piercus) [inline tests/issue-28.js]:
```suggestion
	[0, 1, 2, 3, 4]
```

File: tests/issue-28.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+];
+
+const predictions = [
+	[4, 3, 2, 1, 0], // X, y, w, h
```
Comment (piercus) [inline tests/issue-28.js]:
```suggestion
	[4, 3, 2, 1, 0],
```

**Review by piercus — CHANGES_REQUESTED** [review]
minor changes requested

**Pull request merged**